### PR TITLE
Added some updates to website

### PR DESCRIPTION
**Pull request by bitkidd**
So, I made several updates to website, the bar with search is now fixed. At the same time had to replace Hellobar with a native one, because it has a more predictable behavior (Hellobar changed its position depending on console being opened or not). Now it looks like this: 

alt text

### CODE REVIEW

File: ace [hunk 1]
```diff
@@ -1,5 +1,12 @@
...
+|--------------------------------------------------------------------------
+|   Running Console Commands
+|--------------------------------------------------------------------------
+|
+|  Here we invoke console commands registered under Ace store.
+|
+*/
 const kernel = require('./bootstrap/kernel')
-kernel()
+kernel()
```
Comment (RomainLanz) [inline ace]: This file shouldn't be changed, also there's no new line at the end of the file.

**Review by RomainLanz — CHANGES_REQUESTED** [review]

**Comment by RomainLanz** [comment]
Thanks for this PR ! ❤️ 

You may remove all `bower_components` files that is not needed.

Note that you need to add a new line at the end of every files (it isn't the case).
You should install `editorconfig` plugin in your IDE/TextEditor to configure automatically these thing (https://github.com/adonisjs/adonis-documentation/blob/develop/.editorconfig).

**Comment by thetutlage** [comment]
Also you can please make the bar match the hello bar background color. Which is basically the theme purple color?

**Comment by bitkidd** [comment]
Made all the fixes and changed the color of the topbar, now it will look like this:

img

**Comment by RomainLanz** [comment]
Cool, could you squash to one commit?

**Comment by keeross** [comment]
Hm, I think I rebased it, but it didn't show up here https://github.com/keeross/adonis-documentation/commit/d3f4b60200b3e770e3c95dfab4584233148f4dee
I did something wrong?

**Comment by bitkidd** [comment]
Okay, done, git bugged a bit

**Comment by thetutlage** [comment]
Thanks @keeross 😄

**Pull request merged**